src/test/java/net/mcreator/integration/TestWorkspaceDataProvider.java

```diff
@@ -340,8 +340,13 @@ private static GeneratableElement getExampleFor(ModElement modElement, Random ra
 			fluid.viscosity = 10;
 			fluid.isGas = _true;
 			fluid.generateBucket = !_true;
+			fluid.bucketName = modElement.getName() + " Bucket";
+			fluid.textureBucket = emptyLists ? "" : "itest";
 			fluid.creativeTab = new TabEntry(modElement.getWorkspace(),
 					getRandomDataListEntry(random, ElementUtil.loadAllTabs(modElement.getWorkspace())));
+			fluid.emptySound = new Sound(modElement.getWorkspace(),

```

Comment:
If you allow empty sounds, consider adding null or empty sound on emptyLists false here for better test coverage